templates/trivy/trivy-secret.yaml

```diff
@@ -7,5 +7,6 @@ metadata:
 {{ include "harbor.labels" . | indent 4 }}
 type: Opaque
 data:
-  redis: {{ include "harbor.redisForTrivyAdapter" . | b64enc }}
+  redisURL: {{ include "harbor.redisForTrivyAdapter" . | b64enc }}
+  gitHubToken: {{ .Values.trivy.gitHubToken | quote }}
```

Comment:
Isn't this token needed to be encoded with base64?

Reply:
@danielpacak 

What do you think?

Reply:
Yes, I think I have missed that. Let me double check if that's the case I'll open a small PR to fix it. Thanks @ywk253100 for pointing this out. Good catch!